web-jehlomat/src/screens/OrganizationVerification/Content.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTheme } from '@mui/material/styles';
+import { useMediaQuery } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { IContentData } from './OrganizationVerification';
+import Container from '@mui/material/Container';
+import { useMemo } from 'react';
+import { primary, white } from '../../utils/colors';
+import { Header } from 'Components/Header/Header';
+
+interface IProps {
+  contentData: IContentData | null
```

Comment:
Please consider adding a check to `null` within the component. Now it would render an empty page.

Reply:
yes, this is an idea to render an empty page